featuretools/primitives/base/primitive_base.py

```diff
@@ -156,3 +156,6 @@ def flatten_nested_input_types(input_types):
                 sub_input for input_obj in input_types for sub_input in input_obj
             ]
         return input_types
+
+    def set_series_library(self, series_library: str):
```

Comment:
Is there a reason to have this as a method rather than just setting the attribute directly?